src/main/java/org/jabref/model/util/ListUtil.java

```diff
@@ -0,0 +1,23 @@
+package org.jabref.model.util;
+
+import java.util.function.Predicate;
+
+public class ListUtil {
+    public static <T> boolean anyMatch(Iterable<T> list, Predicate<T> predicate) {
```

Comment:
It's about a factor two faster than the streams solution.

Reply:
Can you just add a JavaDoc, please?

Would it be possible to add a performance test? So that if we replace this by something with promising performance (e.g., Goldman Sach's Collections), we can easily check which one is faster.